### PR TITLE
[improve#Doc] add sign icla guide doc

### PR DESCRIPTION
**Pull request by SnowMoon-Dev**
## Brief change log

1. add sign icla guide doc

## Verify this pull request

Manually verified the change by testing locally

### CODE REVIEW

**Comment by warren830** [comment]
Thank you @SnowMoon-Dev !

File: community/make-contribution/sign-icla-guide.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+```
+secretary@apache.org
+```
+
+template
+
+```
+Hello Apache,
+    I am willing donate the DevLake project to ASF. The attachment is my ICLA information. Github account: https://github.com/xxxx.
```
Comment (klesh) [inline community/make-contribution/sign-icla-guide.md]: The template doesn't seem to be correct.

Comment (SnowMoon-Dev) [reply community/make-contribution/sign-icla-guide.md]: do you have any good advice? @klesh

Comment (SnowMoon-Dev) [reply community/make-contribution/sign-icla-guide.md]: @klesh I asked my friends and based on the replies I changed the email template

**Review by klesh — APPROVED** [review]
LGTM

**Pull request merged**